### DISCUSSION TITLE
push image for relay node

### DISCUSSION THREAD

**Discussion by huang12zheng** [qaul/qaul.net#518] [Ideas]
I found that without documentation and best practices for relay nodes,
Maybe we could add it somewhere?

This is an example that I wrote
https://github.com/huang12zheng/qaul.net/tree/dockerfile

**Comment by brenodt** [comment]
Thanks for the example!
I think it's a nice idea to dockerize qauld and enable self-hosting - it's something we talked about internally as well.
We're investigating a way of doing so, probably by hosting an image over the Docker hub.
I've opened issue #520 to track this.

**Comment by MathJud** [comment]
Dear @huang12zheng 

Thanks a lot for this contribution and the opening of the discussion!

You can find a documentation on how to run a `relay server`, which we call  `Community Node` you can find here:
https://qaul.net/tutorials/community-node/

It's good to have a docker version!
Here a few cents from my side. I think it would be nice to have a few options for the user in the docker-compose:

* a mapping of the user data to the host system
* configurable default user name of the Node
* optional: configurable port

@brenodt thanks for opening the issue.

**Comment by huang12zheng** [comment]
The repo does not use the github action, but may still need CI/CD to push the image

**Comment by MathJud** [comment]
@huang12zheng 
We are looking into that for the CI/CD!

**Comment by brenodt** [comment]
Since this has been fixed, I believe we can close this discussion.
You can find qauld images at the [qaulnet docker hub](https://hub.docker.com/r/qaulnet/qauld).
